src/test/run-pass/issues/issue-61442.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(generators)]
```

Comment:
I really don't like having a bunch of numbered issues in `run-pass` (instead of `compile-pass`) with zero description of what this is supposed to test...

Reply:
I think this test should be moved to the other generator tests wherever they are.

Reply:
src/test/run-pass/generator/